RIRImageResizing/Classes/NSObject/UIImage_RIRResizing_ResizeModes.h

```diff
@@ -0,0 +1,30 @@
+//
+//  UIImage_RIRResizing_ResizeModes.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import <ResplendentUtilities/RUEnumIsInRangeSynthesization.h>
```

Comment:
This should actually go in the #ifndef below. Check other similar files if you want, like `SMAlertView+SMLockedTribeAlertView_buttonOptions.h`.